### PR TITLE
debian: List available firmware updates in gnome-software

### PR DESCRIPTION
**Pull request by starnight**
- Enable fwupd
- Enable libfwupd-dev

Signed-off-by: Jian-Hong Pan <jian-hong@endlessm.com>

https://phabricator.endlessm.com/T29506

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -7,6 +7,7 @@ export DEB_LDFLAGS_MAINT_APPEND = -Wl,-O1 -Wl,--as-needed
 export DEB_LDFLAGS_MAINT_STRIP := -Wl,-Bsymbolic-functions
 
 GS_CONFIGURE_FLAGS = \
+	-Dfwupd=true \
```
Comment (pwithnall) [inline debian/rules]: Please don’t touch the `-Dfwupd` settings here or in the “these are conditionally re-enabled later” section below, as those sections come from upstream Debian packaging, and any changes we make to them make the rebase a little harder.

You need to set `-Dfwupd-true` in the “EOS-specific configuration section” further down.

Comment (starnight) [reply debian/rules]: Thanks for your review!  That makes sense, so I update the commit.  Please review again.  Thank you!

**Comment by pwithnall** [comment]
Converting to a draft because it’s blocked on a few infrastructure improvements needed for firmware signing (https://phabricator.endlessm.com/T29505, https://phabricator.endlessm.com/T29930).

**Comment by wjt** [comment]
oh great jenkins bot, retest this please

**Comment by wjt** [comment]
@starnight could you rebase this change please?

**Comment by starnight** [comment]
> @starnight could you rebase this change please?

Sure!  And ready for review now.

**Pull request merged**